.github/workflows/pegasus.yml

```diff
@@ -50,6 +50,12 @@ jobs:
   k8s-test:
     runs-on: [self-hosted, ubuntu2004]
     steps:
+    - name: Detect the tmate session
+      if: "jobs.k8s-test.steps[*].uses == 'mxschmitt/action-tmate@*'"
```

Comment:
why this is still use the context while others use `grep`

Reply:
My bad. Thanks for catching this.